### PR TITLE
Fixed some grammar and spelling issues, modified sign in form

### PR DESCRIPTION
**Pull request by Brosilio**
There were some grammatical and spelling errors that bothered me, so I fixed them. I think. At least, the ones I noticed.
I also slightly modified the Sign In form to have a more consistent and better look.

Here's what it looks like now:

It is functionally the same as it was before, but now it looks a little better.

### Before submitting a pull-request to GitBucket I have first:

- [x] read the [contribution guidelines](https://github.com/gitbucket/gitbucket/blob/master/.github/CONTRIBUTING.md)
- [ ] rebased my branch over master
- [x] verified that project is compiling
- [x] verified that tests are passing
- [x] squashed my commits as appropriate *(keep several commits if it is relevant to understand the PR)*
- [ ] [marked as closed using commit message](https://help.github.com/articles/closing-issues-via-commit-messages/) all issue ID that this PR should correct

### CODE REVIEW

File: src/main/twirl/gitbucket/core/account/hooks.scala.html [hunk 1]
```diff
@@ -11,9 +11,9 @@
       </div>
       <div class="panel-body">
         <p>
-          Webhooks allow external services to be notified when certain events happen within your repository.
-          When the specified events happen, we’ll send a POST request to each of the URLs you provide.
-          Learn more in <a href="https://github.com/takezoe/gitbucket/wiki/API-WebHook" target="_blank">GitBucket Wiki Webhook Page</a>.
+          These webhooks notify external services when certain events occur within any of your repositories.
+          When any of the specified events occur, GitBucket will send a POST request to all of the endpoints (URLs) you provide.
+          Learn more about this feature on the <a href="https://github.com/takezoe/gitbucket/wiki/API-WebHook" target="_blank">GitBucket Wiki</a>.
```
Comment (SIkebe) [inline src/main/twirl/gitbucket/core/account/hooks.scala.html:16]:
```suggestion
          Learn more about this feature on the <a href="https://github.com/gitbucket/gitbucket/wiki/API-WebHook" target="_blank">GitBucket Wiki</a>.
```

File: src/main/twirl/gitbucket/core/settings/branches.scala.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
         <div class="panel panel-default">
           <div class="panel-heading strong">Default branch</div>
           <div class="panel-body">
-            <p>The default branch is considered the “base” branch in your repository, against which all pull requests and code commits are automatically made, unless you specify a different branch.</p>
+            <p>The default branch is considered the “base” branch in your repository, against which all pull requests and code commits are automatically made (unless you specify a different branch.)</p>
```
Comment (SIkebe) [inline src/main/twirl/gitbucket/core/settings/branches.scala.html:21]:
```suggestion
            <p>The default branch is considered the “base” branch in your repository, against which all pull requests and code commits are automatically made (unless you specify a different branch).</p>
```

File: src/main/twirl/gitbucket/core/settings/danger.scala.html [hunk 3]
```diff
@@ -55,7 +55,7 @@
               <fieldset class="border-top form-group">
                 <label class="strong">Delete repository</label>
                 <div>
-                  Once you delete a repository, there is no going back.
+                  Permanently deletes this repository. This cannot be undone.
```
Comment (SIkebe) [inline src/main/twirl/gitbucket/core/settings/danger.scala.html:58]:
```suggestion
                  Permanently delete this repository. This cannot be undone.
```

File: src/main/twirl/gitbucket/core/settings/hooks.scala.html [hunk 4]
```diff
@@ -12,9 +12,9 @@
         </div>
         <div class="panel-body">
           <p>
-            Webhooks allow external services to be notified when certain events happen within your repository.
-            When the specified events happen, we’ll send a POST request to each of the URLs you provide.
-            Learn more in <a href="https://github.com/takezoe/gitbucket/wiki/API-WebHook" target="_blank">GitBucket Wiki Webhook Page</a>.
+            Webhooks notify external services when certain events occur within this repository.
+            When any of the specified events occur, GitBucket will send a POST request to all of the endpoints (URLs) you provide.
+            Learn more about this feature on the <a href="https://github.com/takezoe/gitbucket/wiki/API-WebHook" target="_blank">GitBucket Wiki</a>.
```
Comment (SIkebe) [inline src/main/twirl/gitbucket/core/settings/hooks.scala.html:17]:
```suggestion
            Learn more about this feature on the <a href="https://github.com/gitbucket/gitbucket/wiki/API-WebHook" target="_blank">GitBucket Wiki</a>.
```

File: src/main/twirl/gitbucket/core/settings/danger.scala.html (hunk 3 above)
Comment (SIkebe) [reply src/main/twirl/gitbucket/core/settings/danger.scala.html:58]: To align with "Transfer Ownership"and "Garbage collection" sections.

File: src/main/twirl/gitbucket/core/account/hooks.scala.html (hunk 1 above)
Comment (takezoe) [reply src/main/twirl/gitbucket/core/account/hooks.scala.html:16]: Oh, old URL was still inside code!

**Comment by takezoe** [comment]
@Brosilio Great thanks for your contribution! Could you take a look at @SIkebe's suggestion and apply them if they make sense to you?

**Comment by Brosilio** [comment]
I applied all the suggestions, they were good. I didn't notice the mistake with the period.

**Pull request merged**